COMPILE_MAKE.TXT

```diff
@@ -1,3 +1,11 @@
+
+# NOTICE
+
+> Please be aware that the Makefile build is deprecated.
+> Use cmake instead.
+
+<hr>
+
 This documentation explains how to compile, install & run Capstone on MacOSX,
 Linux, *BSD & Solaris. We also show steps to cross-compile for Microsoft Windows.
 
```

Comment:
Yes, but I would do it in another PR. I also did not edited or removed the MSVC project files. Because it gets too bloated otherwise.
But opened an issue about it: https://github.com/capstone-engine/capstone/issues/2457